src/XR.tsx

```diff
@@ -106,6 +117,15 @@ function XRManager({
     set(() => ({ foveation }))
   }, [gl.xr, foveation, set])
 
+  useIsomorphicLayoutEffect(() => {
+    try {
+      if (frameRate) session?.updateTargetFrameRate?.(frameRate)
+    } catch (_) {
+      // Framerate not supported or configurable
+    }
+    set(() => ({ frameRate }))
+  }, [session, frameRate, set])
+
```

Comment:
We can possibly make this more helpful by logging `XRSession.supportedFramerates` as per https://www.w3.org/TR/webxr/#xrsession-interface. Would it be useful to use `XRSession.frameRate` as the default though? This would be available through `useXR` and can be procedurally tuned.

Reply:
I'm not sure if it be useful or even correct. Platform might report something as a supported frame rate but fail/ignore the setting or it could be that updating is possible, but no concrete supported framerates are provided